pre_commit/resources/hook-tmpl

```diff
@@ -40,6 +41,7 @@ fi
 {pre_push}
 
 # Run pre-commit
+args="$args --hook-stage $hook_stage"
```

Comment:
Yeah, I think for pre-push this is already the case, you'll just need to add it for pre-commit (or if you make it default to `commit`, you can probably avoid needing to edit args in this file)